src/views/Home.vue

```diff
@@ -28,7 +28,7 @@
           <ion-card-title>Donde llevar los elementos a reciclar</ion-card-title>
         </ion-card-header>
         <ion-card-content> Ingresá y conocé donde llevarlos. </ion-card-content>
-        <ion-button expand="full" color="success" href="/mapa"
+        <ion-button expand="full" color="success" router-link="/tabs/mapa"
```

Comment:
con router link te muestra una animacion y te permite volver hacia atras, href se usa mas para redirigir a paginas externas segun vi

Reply:
```suggestion
        <ion-button expand="full" color="success" router-link="/mapa"
```

Deberia llevar solo /mapa, ya que el /tabs era a modo de ejemplo y en nuestro sistema no es necesario.

Reply:
si, como estaba asi de entrada lo deje por las dudas, despues les saco el prefijo de tabs

Reply:
UPDATE viendo la documentacion, ellos tambien lo hacen con el prefijo de tabs, probe sacarlo y no pude hacer que anduviera, si lo logras genial jaja, pero tampoco me parece gastarse tanto total el usuario ni lo va a ver eso

https://ionicframework.com/docs/vue/navigation#working-with-tabs